layouts/partials/page-meta-links.html

```diff
@@ -0,0 +1,28 @@
+{{ if .Path }}
+{{ $pathFormatted := replace .Path "\\" "/" }}
+{{ $gh_repo := ($.Param "github_repo") }}
+{{ $gh_subdir := ($.Param "github_subdir") }}
+{{ $gh_project_repo := ($.Param "github_project_repo") }}
+{{ $gh_branch := (default "master" ($.Param "github_branch")) }}
+{{ if $gh_repo }}
+<div class="td-page-meta ml-2 pb-1 pt-2 mb-0">
+{{ $editURL := printf "%s/edit/%s/content/%s" $gh_repo $gh_branch $pathFormatted }}
+{{ if and ($gh_subdir) (.Site.Language.Lang) }}
+{{ $editURL = printf "%s/edit/%s/%s/content/%s/%s" $gh_repo $gh_branch $gh_subdir ($.Site.Language.Lang) $pathFormatted }}
+{{ else if .Site.Language.Lang }}
+{{ $editURL = printf "%s/edit/%s/content/%s/%s" $gh_repo $gh_branch ($.Site.Language.Lang) $pathFormatted }}
+{{ else if $gh_subdir }}
+{{ $editURL = printf "%s/edit/%s/%s/content/%s" $gh_repo $gh_branch $gh_subdir $pathFormatted }}
+{{ end }}
+{{ $issuesURL := printf "%s/issues/new?title=%s" $gh_repo (htmlEscape $.Title )}}
+{{ if $gh_project_repo }}
+{{ $project_issueURL := printf "%s/issues/new" $gh_project_repo }}
+<a href="{{ $project_issueURL }}" target="_blank"><i class="fab fa-github fa-fw"></i> {{ T "post_create_project_issue" }}</a>
```

Comment:
I'm not sure we need a link to SUSHI issues in the sidebar -- especially since we have one in the footer.  And the test "Create a project issue" is confusing... which project?

I think it would be better to have a link that said "Report a documentation issue" or something like that which goes to GitHub issues for this site.

Reply:
I kinda like having the link to SUSHI bugs, since most of this documentation does pertain to SUSHI, and I know personally I've never looked too close at footer links. So I'd like to keep that unless you really dislike it. But I agree that the name should be changed to reference SUSHI, and I can add a "report a documentation issue" link as well.

Reply:
Sounds like a reasonable compromise.